src/main/webapp/app/exercises/shared/dashboards/tutor/exercise-assessment-dashboard.component.ts

```diff
@@ -330,6 +330,21 @@ export class ExerciseAssessmentDashboardComponent implements OnInit {
         }
     }
 
+    get yourStatusTitle(): string {
+        switch (this.tutorParticipationStatus) {
+            case TutorParticipationStatus.TRAINED:
+                // If we are in 'TRAINED' state, but never really "trained" on example submissions, display the
+                // 'REVIEWED_INSTRUCTIONS' state text instead.
+                if (!this.exercise.exampleSubmissions || this.exercise.exampleSubmissions.length == 0) {
```

Comment:
GitHub Actions complains:
```suggestion
                if (!this.exercise.exampleSubmissions || this.exercise.exampleSubmissions.length === 0) {
```

Reply:
Thank you @FrankeLukas. Done ✅